articles/api/authentication/_login.md

```diff
@@ -229,59 +195,41 @@ GET https://${account.namespace}/authorize?
 ```
 
 ```javascript
-<script src="${auth0js_url}"></script>
+// Script uses auth0.js v8. See Remarks for details.
```

Comment:
Yes, `buildAuthorizeUrl` is what we need to use. It takes the params that an authorize request would need.

```
webAuth.client.buildAuthorizeUrl({
  clientID: 'YOUR_CLIENT_ID', // string
  responseType: 'token', // code or token
  redirectUri: 'http://example.com',
});
```

Something like this? Also will want to recommend setting a `state` probably.